sdk/dotnet/Pulumi.Kubernetes.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi" Version="1.12.1-preview" />
+    <PackageReference Include="Pulumi" Version="2.0.0" />
```

Comment:
Is that intentionally a specific version and not `2.*`?

Reply:
It is not - changing now!